lib/pr_checker.js

```diff
@@ -132,12 +133,16 @@ class PRChecker {
    */
   getWait(now) {
     const createTime = new Date(this.pr.createdAt);
-    const timeLeft = WAIT_TIME - Math.ceil(
+    const hoursFromCreateTime = Math.ceil(
```

Comment:
This seems rather complicated. I believe we don't have to special case if we combine `checkReviews()` and `checkPRWait()` so that the number of approvals and wait times are checked together with conditionals instead of independently, the logic would be much clearer that way.

Reply:
(The two methods are separated only because previously a human would check these two conditions separately, but now there is a relationship between the two we should do what a human brain would do here instead and combine the logic.)